src/pages/CreateEvent/EventForm.tsx

```diff
@@ -163,230 +232,373 @@ const EventForm: React.FC = () => {
-                  rules={{ required: "Start date is required" }}
-                  render={({ field }) => (
-                    <input
-                      type="date"
-                      {...field}
-                      className={`form-control ${
-                        errors.startDate ? "error" : ""
-                      }`}
-                    />
-                  )}
-                />
+          <label htmlFor="theme">Theme<sup>*</sup></label>
+          <div
+            className={`border-2 rounded-lg border-solid flex gap-4 p-2 space-x-2 ${
+              errors.disciplines ? "border-red-500" : "border-black"
+            }`}
+          >
+            {selectedThemes.map((theme) => (
+              <div
+                key={theme}
+                className="bg-yellow-400 flex items-center px-6 py-1 rounded-full leading-[115.645%]"
+              >
+                {theme}
+                <button
+                  onClick={() => removeTheme(theme)}
+                  className="ml-4 text-black"
+                >
+                  ✕
+                </button>
               </div>
+            ))}
+             <select
+              onChange={handleThemesChange}
+              className={`focus:outline-none w-full theme-select ${selectedThemes.length === 3 && 'appearance-none'}`}
+              disabled={selectedThemes.length === 3}
+            >
+              <option value={""}>{selectedThemes.length == 0 && 'Select up to 3 themes'}</option>
+              {allThemes
+                .filter((theme) => !selectedThemes.includes(theme))
+                .map((theme) => (
+                  <option key={theme} value={theme}>
+                    {theme}
+                  </option>
+                ))}
+            </select>
+          </div>
+          {errors.themes && <p className="error">{errors.themes.message}</p>}
+        </div>
 
-              <div className="time-input">
-                <ClockIcon className="icon" />
-                <Controller
-                  name="startTime"
-                  control={control}
-                  rules={{ required: "Start time is required" }}
-                  render={({ field }) => (
-                    <input
-                      type="time"
-                      {...field}
-                      className={`form-control ${
-                        errors.startTime ? "error" : ""
-                      }`}
-                    />
-                  )}
-                />
-              </div>
+        <div className="form-group">
+          <label htmlFor="eventDuration">Event Duration<sup>*</sup></label>
+
+          <div className="sub-section">
+            <span>Start<sup>*</sup></span>
+            <div className="date-input-container">
+              <Controller
+                name="startDate"
+                control={control}
+                rules={{ required: "Start date is required" }}
+                render={({ field }) => (
```

Comment:
should black out (make unselectable) the past dates and dates less than 2 weeks from current date

Reply:
Done